imports/startup/server/index.coffee

```diff
@@ -44,196 +223,45 @@ escape = (text)->
-        ?resolvedTerm rdfs:label ?termLabel
-        FILTER (?dateTime > "#{escape(baseYear)}-01-01T00:00:00+00:01"^^xsd:dateTime)
-      }
-      GROUP BY ?resolvedTerm
-      ORDER BY DESC(?count)
-      LIMIT 20
-      """
-    makeRequest(query)
-
+    response = makeRequest(getFrequentlyMentionedAgentsQuery())
   'getRecentMentions': (agent) ->
-    query = prefixes + """
-      SELECT DISTINCT
-        ?phrase_text ?p_start
-        ?t_start ?t_end
-        ?source ?date
-      WHERE {
-          ?phrase anno:selected-text ?phrase_text
-          ; anno:start ?p_start
-          ; anno:end ?p_end
-          ; dep:ROOT ?noop
-          ; anno:contains ?target
-          .
-          {
-              ?target anno:label "#{escape(agent)}"
-          } UNION {
-              ?resolvedTarget dc:relation ?target
-              ; rdfs:label "#{escape(agent)}"
-          } .
-          ?target anno:start ?t_start
-          ; anno:end ?t_end
-          ; anno:source_doc ?source
-          .
-          ?source pro:post/pro:date ?p_date .
-          OPTIONAL { ?source  pro:date  ?a_date }
-          BIND(coalesce(?a_date, ?p_date) AS ?date)
-      }
-      ORDER BY DESC(?date) DESC(?source) ASC(?t_start)
-      LIMIT 10
-      """
-    response = makeRequest(query)
+    response = makeRequest(getRecentMentionsQuery(agent))
     response.results.bindings.map(castBinding)
-
-  'getFrequentDescriptors': (ia) ->
-    query = prefixes + """
-      SELECT
-        ?selText
-        (group_concat(DISTINCT ?article; separator = "::") AS ?articles)
-        (count(DISTINCT ?article) as ?count)
-      WHERE {
-          ?dep_rel rdf:type anno:dependency_relation .
-          VALUES ?dep_rel { dep:amod dep:nmod }
-          ?parent anno:min_contains ?target
-              ; ?dep_rel ?descriptor
-              ; anno:source_doc ?article
-              .
-          ?descriptor anno:start ?d_start
-              ; anno:end ?d_end
-              ; anno:selected-text ?rawSelText
-              ; anno:root/anno:pos ?pos
-              .
-          FILTER (?pos NOT IN ("X", "PUNCT"))
-          ?target anno:category "diseases"
-              ; anno:start ?t_start
-              ; anno:end ?t_end
-              ; ^dc:relation ?rel
-              .
-          	?rel rdfs:label "#{escape(ia)}"
-          FILTER ( ?d_end <= ?t_start || ?t_end <= ?d_start )
-          BIND(lcase(?rawSelText) as ?ranCaseSelText)
-          #remove leading and trailing whitespace, and new line characters
-          BIND(replace(?ranCaseSelText,'^ +| +$|\\n', '') AS ?selText)
-      }
-      GROUP BY ?selText
-      HAVING (?count > 0)
-      ORDER BY DESC(?count)
-      """
-    response = makeRequest(query)
+  'getFrequentDescriptors': (agent) ->
+    response = makeRequest(getFrequentDescriptorsQuery(agent))
     response.results.bindings.map(castBinding)
+
+api = new Restivus
+  useDefaultAuth: true
+  prettyJson: true
+
+api.addRoute 'frequentDescriptors/:ia',
```

Comment:
We should replace the "ia" variable name with something a bit more verbose like "term"